app/views/collection_associations/index.html.erb

```diff
@@ -32,28 +32,28 @@
             <th>Version</th>
           </tr>
         </thead>
-        <tbody>
 
-        <% if @associations.blank? %>
-          <tr>
-            <td colspan="5">
-              <%= "No Collection #{association_text} found." %>
-            </td>
-          </tr>
-        <% else %>
-          <% @associations.each do |collection| %>
-            <tr class='<%= cycle("alt", "") %>'>
-              <td class="align-c"><%= check_box_tag('selected_collections[]', collection.fetch('meta', {})['concept-id'], false, class: 'selected_collections') %></td>
-              <td><%= link_to collection.fetch('umm', {}).fetch('entry-title'), collection_path(collection.fetch('meta', {}).fetch('concept-id')) %></td>
-              <td><%= collection.fetch('umm', {}).fetch('short-name') %></td>
-              <td><%= collection.fetch('umm', {}).fetch('version-id') %></td>
+        <tbody>
+          <% if @associations.blank? %>
+            <tr>
+              <td colspan="4">
+                No Collection Associations found.
+              </td>
             </tr>
+          <% else %>
+            <% @associations.each do |collection| %>
+              <tr class='<%= cycle("alt", "") %>'>
+                <td class="align-c"><%= check_box_tag('selected_collections[]', collection.fetch('meta', {})['concept-id'], false, class: 'selected_collections') %></td>
+                <td><%= link_to collection.fetch('umm', {}).fetch('entry-title'), collection_path(collection.fetch('meta', {}).fetch('concept-id')) %></td>
```

Comment:
I know you didn't actually write it, but the double `fetch` could be `dig` since the trailing `fetch`'s don't have the optional `{}` parameter which means they'll just return `nil` if the key is not found.

Reply:
I agree, but as you mentioned, it was already there and I think these changes were either things being moved around or indented. If I was actually modifying any of this I would have made the change (I think I did in some other places I was working in).